src/components/elements/HeroDesktop.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const heroStyle = `
+  flex 
+  justify-between 
+  items-end 
+  h-48
+  mb-20
+  pb-5 
+  pl-20
+  pr-32  
+  bg-green-100
+`
+
+export default function HeroDesktop() {
+  return (
+    <>
```

Comment:
We don't need the fragments here, since there's only 1 top level parent (the div), also the div should likely be a "section"

Reply:
Done!